src/Containers/Doctors/_doctors.scss

```diff
@@ -34,16 +39,16 @@
   }
 }
 
-@media screen and (max-width: 732px) {
+@media screen and (max-width: 755px) {
   .all-drs {
-    width: fit-content;
+
     display: grid;
     grid-template-columns: repeat(1, 1fr);
     grid-gap: 20px;
   }
 }
 
-@media screen and (max-width: 725px) {
+@media screen and (max-width: 320px) {
   .all-drs {
     display: flex;
     flex-direction: column;
```

Comment:
We'll probably want to adjust this when we get a chance. It looks like the information starts to overflow the card when a lot of insurances are listed on a small screen